dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -280,14 +280,21 @@ public GHBlob tryRetrievingBlob(GHRepository repo, String path, String branch)
     public void modifyOnGithub(GHContent content,
                                String branch, String img, String tag,
                                String customMessage, String ignoreImageString) throws IOException {
+        modifyContentOnGithub(content, branch, img, tag, customMessage, ignoreImageString);
+    }
+
+    protected boolean modifyContentOnGithub(GHContent content,
```

Comment:
Utility classes are typically labeled as `final` and have private constructors to prevent extension.  We didn't do that in the case of this class.  As such I would make these methods private or package private if possible to prevent customers taking a dependency on them.